crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -3,75 +3,100 @@
+        update_epoch: u64
     }
 
     struct CommitteeMember has drop, store {
         /// The Sui Address of the validator
         sui_address: address,
         /// The public key bytes of the bridge key
         bridge_pubkey_bytes: vector<u8>,
-        /// Voting power
-        voting_power: u64,
+        /// Voting power (stake amount)
+        stake_amount: u64,
         /// The HTTP REST URL the member's node listens to
         /// it looks like b'https://127.0.0.1:9191'
         http_rest_url: vector<u8>,
         /// If this member is blocklisted
         blocklisted: bool,
     }
 
+    struct CommitteeMemberRegistration has drop, store {
+        /// The Sui Address of the validator
+        sui_address: address,
+        /// The public key bytes of the bridge key
+        bridge_pubkey_bytes: vector<u8>,
+        /// The HTTP REST URL the member's node listens to
+        /// it looks like b'https://127.0.0.1:9191'
+        http_rest_url: vector<u8>,
+    }
+
     public(friend) fun create(ctx: &TxContext): BridgeCommittee {
         assert!(tx_context::sender(ctx) == @0x0, ENotSystemAddress);
-        // Hardcoded genesis committee
-        // TODO: change this to real committe members
-        let members = vec_map::empty<vector<u8>, CommitteeMember>();
-
-        let bridge_pubkey_bytes = hex::decode(b"029bef8d556d80e43ae7e0becb3a7e6838b95defe45896ed6075bb9035d06c9964");
-        vec_map::insert(&mut members, bridge_pubkey_bytes, CommitteeMember {
-            sui_address: address::from_u256(1),
-            bridge_pubkey_bytes,
-            voting_power: 10,
-            http_rest_url: b"https://127.0.0.1:9191",
-            blocklisted: false
-        });
-
-        let bridge_pubkey_bytes = hex::decode(b"033e99a541db69bd32040dfe5037fbf5210dafa8151a71e21c5204b05d95ce0a62");
-        vec_map::insert(&mut members, bridge_pubkey_bytes, CommitteeMember {
-            sui_address: address::from_u256(2),
-            bridge_pubkey_bytes,
-            voting_power: 10,
-            http_rest_url: b"https://127.0.0.1:9192",
-            blocklisted: false
-        });
-
+        // Default signature threshold
         let thresholds = vec_map::empty();
-        vec_map::insert(&mut thresholds, message_types::token(), 10);
-        BridgeCommittee { members, thresholds }
+        vec_map::insert(&mut thresholds, message_types::token(), 50);
+        vec_map::insert(&mut thresholds, message_types::committee_blocklist(), 50);
+        vec_map::insert(&mut thresholds, message_types::emergency_op(), 50);
+        vec_map::insert(&mut thresholds, message_types::update_asset_price(), 50);
+        vec_map::insert(&mut thresholds, message_types::update_bridge_limit(), 50);
+        BridgeCommittee {
+            members: vec_map::empty(),
+            stake_thresholds_percentage: thresholds,
+            member_registration: vec_map::empty(),
+            total_member_stake: 0,
+            update_epoch: 0,
```

Comment:
what does this do?

Reply:
this record the epoch when the current committee was created

Reply:
will add more comments